cdap-docs/admin-manual/source/installation/azure-hdinsight.rst

```diff
@@ -0,0 +1,156 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+
+.. :section-numbering: true
+
+.. _admin-installation-azure-hdinsight:
+
+=========================================
+Installation on Microsoft Azure HDInsight
+=========================================
+
+Introduction
+============
+
+This section describes installing CDAP on `Microsoft Azure HDInsight
+<https://azure.microsoft.com/en-us/services/hdinsight/>`__ using
+Microsoft's website to:
+
+- Create an HBase cluster;
```

Comment:
it's an HDInsight cluster of type HBase... i'd suggest "Create an HDInsight HBase cluster" or similar.